### PR TITLE
Jellyfin: new, 10.7.7

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

Jellyfin: new, 10.7.7

Package(s) Affected
-------------------

Jellyfin: 10.7.7

Security Update?
----------------

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

### CODE REVIEW

**Comment by eatradish** [comment]
LGTM on AMD64:

Screenshot_20211208_021656

Screenshot_20211208_021618

Screenshot_20211208_021530

**Comment by eatradish** [comment]
Systemd service also LGTM

File: extra-devel/aspnetcore-runtime-6.0/spec [hunk 1]
```diff
@@ -9,3 +9,4 @@ CHKSUMS__ARMV7HF="sha512::36be738bb40a0cadacd4531c3597a25fd45deb7c48090ffb61c79a
 
 SUBDIR=.
 CHKUPDATE="html::url=https://dotnetcli.azureedge.net/dotnet/release-metadata/releases-index.json;pattern=\"latest-runtime\": \"(6\..+?)\""
+CHKSUM="sha256::07dd14194e37c839a9580a212f9fd47131d443058f399177c3747b56fad35948"
```
Comment (KexyBiscuit) [inline extra-devel/aspnetcore-runtime-6.0/spec]: Why adding these checksums?

Comment (MingcongBai) [reply extra-devel/aspnetcore-runtime-6.0/spec]: Ditto.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**